Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -162,9 +162,21 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
         CommentsViewModel.postCommentProducer
       )
 
-    let repliesAndTotalCount = replies.withLatestFrom(totalCountProperty.signal)
+    let errorLoadingMoreReplies =
+      error
+        .withLatestFrom(totalCountProperty.signal)
+        .combineLatest(with: project)
+        .map { errorAndTotalCount, project -> (([Comment], Int), Project) in
+          let (_, totalCount) = errorAndTotalCount
+          return (([], totalCount), project)
```

Comment:
Empty array, similar to what we're doing in `CommentsViewModel` for the same situation.

Reply:
This doesn't work. Going to change it.